src/metric-engine/src/metadata_region.rs

```diff
@@ -509,6 +534,10 @@ impl MetadataRegion {
             .await
             .context(MitoWriteOperationSnafu)?;
 
+        for (logical_region_id, _) in logical_regions {
+            self.add_logical_region(logical_region_id).await?;
+        }
```

Comment:
Is it safe to add all logical regions without considering existence? When do we call `add_logical_regions()`? Maybe more documents are needed.